index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>youtube</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="youtube.css">
+        <link rel="icon"  href="youtube.png">
+    </head>
+    <body>
+     
+        <div id="nav-bar">
+            <div id="nav-menu"><img src="menu.png"/></div>
+            <div id="logo">
+                <div id="logo1"><img  src="youtube.png"/></div>
+                <div id="logo2">YouTube</div>
+            </div>
+            <div id="nav-search">
+                <input type="search" placeholder="search"/><img src="loupe.png"/>
+            </div>
+        </div>
+    <div id="container">
+        
+        <div id="displyed_video_container">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/lJIrF4YjHfQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            <h3>title of this video </h3>
+           
+        
+        </div> 
+        <div id="side-bar"> 
```

Comment:
please use <aside> instead of div,it's better to use semantic HTML tags

Reply:
Please ensure that all your comments are clear and easy to understand.
Kindly make a necessary correction to the comment to state clearly the changes requested. What do I use instead of the div tag?